server/app/views/ProgramImageUtils.java

```diff
@@ -0,0 +1,75 @@
+package views;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static j2html.TagCreator.img;
+
+import com.google.inject.Inject;
+import j2html.tags.specialized.ImgTag;
+import java.util.Locale;
+import java.util.Optional;
+import play.mvc.Http;
+import services.cloud.PublicFileNameFormatter;
+import services.cloud.PublicStorageClient;
+import services.program.ProgramDefinition;
+import services.settings.SettingsManifest;
+import views.style.StyleUtils;
+
+/** Utility class for rendering program images. */
+public final class ProgramImageUtils {
+  private final PublicStorageClient publicStorageClient;
+
+  private final SettingsManifest settingsManifest;
+
+  @Inject
+  public ProgramImageUtils(
```

Comment:
question/suggestion (non-blocking): Do you need unit tests for `ProgramImageUtils` or are these covered elsewhere?

Reply:
I was thinking that they're partially covered by screenshot tests, but this is a great suggestion. Added tests.